app/controllers/roles_controller.rb

```diff
@@ -17,4 +11,13 @@ def update
     p params
     return render json: { test: 'hoho' }
   end
+
+  private
+
+  def user_roles
+    existing = current_user.roles
+    remainder = (1..(3-existing.count)).map{ |_| Role.new }
```

Comment:
Haha... actually the best reason to use Vim is that VimGolf (the fewest keystrokes wins) is the world's best computer game.